redis/client.py

```diff
@@ -413,6 +413,34 @@ def parse_slowlog_get(response, **options):
     } for item in response]
 
 
+def parse_stralgo(response, **options):
+    """
+    Parse the response from `STRALGO` command.
+    Without modifiers the returned value is string.
+    When LEN is given the command returns the length of the result
+    (i.e integer).
+    When IDX is given the command returns a dictionary with the LCS
+    length and all the ranges in both the strings, start and end
+    offset for each string, where there are matches.
+    When WITHMATCHLEN is given, each array representing a match will
+    also have the length of the match at the beginning of the array.
+    """
+    if options['len']:
+        return int(response)
+    if options['idx']:
```

Comment:
```suggestion
    if options.has_key('idx'):
```

Reply:
Maybe options.get('idx', False) will be better here @chayim ?